pkg/gateway/server/oauth_apps.go

```diff
@@ -308,6 +308,10 @@ func (s *Server) refreshOAuthApp(apiContext api.Context) error {
 		return fmt.Errorf("failed to make token request: %w", err)
 	}
 	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+	if app.Spec.Manifest.Type != types2.OAuthAppTypeGoogle &&
+		app.Spec.Manifest.Type != types2.OAuthAppTypePagerDuty {
+		req.SetBasicAuth(url.QueryEscape(app.Spec.Manifest.ClientID), url.QueryEscape(app.Spec.Manifest.ClientSecret))
```

Comment:
Do other Oauth Apps need this?

Reply:
yeah but that was for exchanging the auth code for access token, refresh token exchange probably doesn't require this, @thedadams can you confirm?

Reply:
My understanding is that this addition isn't necessary.

Reply:
Removed, it's not needed for the refresh tokens.